src/routes/write-exercise/__tests__/WriteExerciseScreen.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import { mockUseLoadFromEndpointWitData } from '../../../testing/mockUseLoadFromEndpoint'
+import React from 'react'
+import { fireEvent, render } from '@testing-library/react-native'
+import WriteExerciseScreen from '../WriteExerciseScreen'
+import createNavigationMock from '../../../testing/createNavigationPropMock'
+import { RouteProp } from '@react-navigation/native'
+import { RoutesParamsType } from '../../../navigation/NavigationTypes'
+import { DocumentTypeFromServer } from '../../../hooks/useLoadDocuments'
+import labels from '../../../constants/labels.json'
+import { ReactTestInstance } from 'react-test-renderer'
+
+jest.mock('../../../components/AudioPlayer', () => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const Text = require('react-native').Text
+  return () => <Text>AudioPlayer</Text>
+})
+
+describe('WriteExerciseScreen', () => {
+  const testDocuments: DocumentTypeFromServer[] = [
+    {
+      audio: '',
+      word: 'Arbeitshose',
+      id: 1,
+      article: 2,
+      document_image: [{ id: 1, image: 'Arbeitshose' }],
+      alternatives: []
+    },
+    {
+      audio: '',
+      word: 'Arbeitsschuhe',
+      id: 2,
+      article: 4,
+      document_image: [{ id: 2, image: 'Arbeitsschuhe' }],
+      alternatives: []
+    }
+  ]
+
+  const navigation = createNavigationMock()
+  const route: RouteProp<RoutesParamsType, 'WriteExercise'> = {
+    key: '',
+    name: 'WriteExercise',
+    params: {
+      extraParams: {
+        disciplineID: 0,
+        disciplineIcon: 'Icon',
+        disciplineTitle: 'Title',
+        exercise: 4,
+        exerciseDescription: 'Description',
+        trainingSet: 'Set',
+        trainingSetId: 0,
+        level: 0
+      }
+    }
+  }
+  it('should allow to skip an exercise and try it out later', async () => {
+    mockUseLoadFromEndpointWitData(testDocuments)
+    // @ts-expect-error
```

Comment:
Please add a comment why we expect an error here

Reply:
@SteffiStuffel what should I put in the comment?

Reply:
just add ` //@ts-expect-error because typescript does not know FiberNode`